packaging/CMakeLists.txt

```diff
@@ -23,12 +23,18 @@ set(Halide_INSTALL_TOOLSDIR "${CMAKE_INSTALL_DATADIR}/tools"
 
 target_sources(Halide_Generator INTERFACE $<INSTALL_INTERFACE:${Halide_INSTALL_TOOLSDIR}/GenGen.cpp>)
 
-install(TARGETS Halide Halide_Generator Halide_LanguageOptions
+set(extra_targets_to_export )
+if (WITH_PYTHON_BINDINGS)
+    set(extra_targets_to_export Halide_PyStubs)
+endif ()
```

Comment:
This should go with the `Halide_Python` install rule in `python_bindings/src/halide/CMakeLists.txt`

Reply:
Will do

Reply:
OK, when I do that, I don't get Halide::PyStubs redirected at all in the install -- do we need to add `Halide_Python` here instead?

Reply:
So this change I couldn't make work at all. Either I misunderstand or, uh, something.

Reply:
I'll take a look